components/console-backend-service/internal/domain/k8s/resourcequota_service.go

```diff
@@ -126,9 +126,9 @@ func (svc *resourceQuotaService) ListStatefulSets(namespace string) ([]*apps.Sta
 	return result, nil
 }
 
-func (svc *resourceQuotaService) ListPods(namespace string, labelSelector map[string]string) ([]v1.Pod, error) {
+func (svc *resourceQuotaService) ListPods(namespace string, envLabelSelector map[string]string) ([]v1.Pod, error) {
```

Comment:
It's just labelSelector, it is not related any env label

Reply:
renamed